python/TestHarness/TestHarness.py

```diff
@@ -946,17 +946,17 @@ def checkForRaceConditionOutputs(self, testers, dirpath):
             name_to_object[tester.getTestName()] = tester
 
         # Now build up our tester dependencies
-        for tester in testers:
-            # Now we need to see which dependencies are real
-            # We don't really care about skipped tests, heavy tests, etc.
-            for name in tester.getPrereqs():
-                if not name_to_object[name].getRunnable():
-                    tester.setStatus('skipped dependency', tester.bucket_skip)
+        try:
+            for tester in testers:
+                # Now we need to see which dependencies are real
+                # We don't really care about skipped tests, heavy tests, etc.
+                for name in tester.getPrereqs():
+                    if not name_to_object[name].getRunnable():
```

Comment:
If there is a typo (a test a user says should be a prereq, but isn't in the list of testers), then that object does not exist in name_to_object dictionary:
```python
name_to_object[name] == None
```